bigtable/client/row_reader_test.cc

```diff
@@ -135,6 +135,25 @@ class BackoffPolicyMock : public bigtable::RPCBackoffPolicy {
   }
 };
 
+class MetadataHolderMock : public bigtable::RPCMetadataHolder {
```

Comment:
Why is this class needed?  And you do know that because `clone()` is not virtual only the implementations in the base class get called, right?